core/shared_queue.js

```diff
@@ -178,6 +178,12 @@ SharedQueue Binary Layout
     return Deno.core.send(opId, control, zeroCopy);
   }
 
+  function refreshOpsMap() {
+    const opsMapBytes = dispatch(0, []);
+    const opsMapJson = String.fromCharCode.apply(null, opsMapBytes);
+    Deno.core.opsMap = JSON.parse(opsMapJson);
```

Comment:
Maybe just call this Deno.core.ops ?

Reply:
Removed completely. Please check out `Deno.core.initOps`